scaling_performance/cns_guidance.adoc

```diff
@@ -0,0 +1,82 @@
+[[scaling-performance-cns-guidance-for-databases]]
+= CNS Guidance For Databases
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[CNS-Guidance-For-Databases]]
+== CNS Guidance for Databases
+In this document will be listed best practices and guidance we recommend 
+for case when CNS ( Container Native Storage ) is used as storage for applications. 
+
+[[Tested-Applciations]]
+== Tested Applications 
+In OCP v3.10 exensive testing was done with (no)SQL databases
+
+-  Postgresql SQL
+-  Mongodb noSQL 
+
+As storage for databases were used storage originating from CNS storage cluster. 
```

Comment:
Not sure what this sentence means. Perhaps:
The storage for these databases originated from a CNS storage cluster.